spec/features/polls/voter_spec.rb

```diff
@@ -4,19 +4,25 @@
 
   context "Origin" do
 
-    let(:poll) { create(:poll, :current) }
+    let(:poll) { create(:poll, :current, starts_at: "2017-12-01", ends_at: "2018-02-01") }
     let(:question) { create(:poll_question, poll: poll) }
     let(:booth) { create(:poll_booth) }
     let(:officer) { create(:poll_officer) }
     let(:admin) { create(:administrator) }
     let!(:answer_yes) { create(:poll_question_answer, question: question, title: 'Yes') }
     let!(:answer_no) { create(:poll_question_answer, question: question, title: 'No') }
 
+    before do
+      allow(Date).to receive(:current).and_return Date.new(2018,1,1)
+      allow(Date).to receive(:today).and_return Date.new(2018,1,1)
+      allow(Time).to receive(:current).and_return Time.zone.parse("2018-01-01 12:00:00")
+    end
+
     background do
       create(:geozone, :in_census)
-      create(:poll_shift, officer: officer, booth: booth, date: Date.current, task: :vote_collection)
+      create(:poll_shift, officer: officer, booth: booth, date: "2018-01-01", task: :vote_collection)
```

Comment:
Couldn't the `Date.current` still be used to set the `date` attribute?

Reply:
Definitely yes, I didn't realised before. I've changed it.